modules/persistent-storage-local-discovery.adoc

```diff
@@ -16,6 +16,13 @@ For more information about the support scope of Red Hat Technology Preview featu
 
 Use the following procedure to automatically discover local devices, and to automatically provision local volumes for selected devices.
 
+[WARNING]
+====
+Use `LocalVolumeSet` with caution.
+When you automatically provision PVs from local disks, the local PVs might claim all devices that match.
+If you are using `LocalVolumeSet`, the Local Storage Operator should be the only entity managing local devices on the node.
```

Comment:
Is there stronger wording we can use over "should"? Perhaps "must", or a variation of this:
```
If you are using a `LocalVolumeSet` object, make sure the Local Storage Operator is the only entity managing local devices on the node.
```

Reply:
Also, please add each of these sentences in the admonition on a line together; we try to avoid new lines unless starting a new paragraph.